PHP tabel number/script.php

```diff
@@ -0,0 +1,8 @@
+<?php    
```

Comment:
@humarh-dharnarh - move the file inside numbers directory, also rename the file to print_tables.php

Reply:
Okay, am on it